source/javascripts/directives/multi_select.js

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
Yeah, you are right – I will rewrite it to Coffeescript and clean it up a little bit when I am in the train to Gänserndorf in half an hour ;-)